source/manual/global-banner.html.md

```diff
@@ -26,6 +26,18 @@ In [`app/views/components/_global_bar.html.erb`]:
    not disappear after being seen 3 times.
 1. [Deploy Static]. See the [guidance on deploying Static](/manual/deploy-static.html).
 
+### Timed Updates
+
+If you now about banner changes in advance and need the updates to show at a
```

Comment:
Typo: should be "know" rather than "now"

Reply:
Thanks! Fixed.